src/test/bloom_tests.cpp

```diff
@@ -138,11 +138,11 @@ BOOST_AUTO_TEST_CASE(bloom_match)
     BOOST_CHECK_MESSAGE(filter.IsRelevantAndUpdate(tx), "Simple Bloom filter didn't match output address");
 
     filter = CBloomFilter(10, 0.000001, 0, BLOOM_UPDATE_ALL);
-    filter.insert(COutPoint(TxidFromString("0x90c122d70786e899529d71dbeba91ba216982fb6ba58f3bdaab65e73b7e9260b"), 0));
+    filter.insert(COutPoint(Txid::FromHex("90c122d70786e899529d71dbeba91ba216982fb6ba58f3bdaab65e73b7e9260b").value(), 0));
```

Comment:
nit: might want to extract to avoid duplicating it 4 and 22 lines below (would make the bloom mistmatch slightly more obvious)

Reply:
going to leave as is to minimize the diff since the bloom tests are not relevant to this PR